oregon-b/snippets-admin/jupyter-cron.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: snippets-jupyter-cron
+  namespace: snippets-admin
+spec:
+  schedule: "@daily"
```

Comment:
I's under the impression that this will be a gitlab job?

Reply:
I removed the k8s cron job from this PR and replaced it with scheduled gitlab jobs in mozmeao/snippets-data#6